sdk/core/azure-core/test/ut/logger.cpp

```diff
@@ -220,7 +221,7 @@ TEST(Logging, Message)
   }
   catch (...)
   {
-    SetLogListener(nullptr);
+    Logger::SetListener(nullptr);
```

Comment:
Might be worth it to test Http Request and Response samples
- Request with allowed and hidden query and headers
- Response with allowed and hidden headers
- Request with no query parameters or headers
- Request/Response with no headers

Reply:
Tests are covered by #271, and will be added later soon.

Reply:
yes, I am not asking to test the other policies, I am asking to test the logger when the message is an Http request and response.  But I think I got what you said